src/services/policiesService.js

```diff
@@ -50,26 +50,33 @@ const getQuestions = async (params) => {
 };
 
 const getPolicyResults = async (body) => {
-  const preguntas = body.answers.map(function(value) {
-    return value.questionId;
+  const arrayPreguntas = body.answers.map(function (value) {
+    return [value.questionId, value.answerId];
   });
 
-  const respuestas = body.answers.map(function(value) {
-    return value.answerId;
-  });
+  let query = `SELECT a.org_politica_id, a.alias, count(*) AS total FROM (SELECT a.*, b.alias FROM partido_x_respuesta a, partidos_alias b WHERE (codPregunta, codRespuesta) IN (VALUES ?) AND a.org_politica_id = b.id) a GROUP BY a.org_politica_id ORDER BY total DESC`;
 
-  let query = `SELECT b.alias AS partido, COUNT(*) AS total FROM (SELECT * FROM partido_x_respuesta WHERE codPregunta IN (?) AND codRespuesta IN (?)) a INNER JOIN partidos_alias b GROUP BY b.alias ORDER BY total DESC`;
-  let response = await db.query(query, [preguntas, respuestas]);
+  let responsePreguntaPartido = await db.query(query, [arrayPreguntas]);
 
-  let partyResults = response.reduce(function(r, a) {
-    const { partido, total } = a;
+  let queryPresidentes = "SELECT * FROM candidato WHERE cargo_nombre LIKE '%PRESIDENTE%'";
```

Comment:
@dmarquinah creo que no es necesario traer todo, sino solo hoja_vida_id y lo que aparece en [figma](https://www.figma.com/file/dvjQTP1M7jaqKux81qfjSx/Votu_UI?node-id=750%3A3584), o sea nombres básicamente (la foto también deriva de hoja_vida_id), el resto del detalle que requiere la siguiente pantalla es el mismo llamado de candidato de siempre, y se hace en la siguiente pantalla con el hoja_vida_id.

Reply:
Got it, no sabía exactamente si ese era el comportamiento, por lo que usé el default en el select (*).
Entonces solo será necesario el id, nombres y foto cierto?

Reply:
sí, sólo eso 👍  (en congreso mandamos más cosas porque tenemos los filtros)